src/refs.bib

```diff
@@ -11,8 +11,8 @@ @INPROCEEDINGS{Knutsson1993
 
 @ARTICLE{2016A&A...594A..13P,
    author = {{Planck Collaboration} and {Ade}, P.~A.~R. and {Aghanim}, N. and
-	{Arnaud}, M. and {Ashdown}, M. and {Aumont}, J. and {Baccigalupi}, C. and
```

Comment:
What's up with the indentation changes here? I am not familiar with the `bib` format.

Reply:
I think my editor automatically changed tabs to spaces.